using-mobile-extensions/adobe-journey-optimizer/android-messaging-push-improvements.md

```diff
@@ -0,0 +1,102 @@
+
+##### To get started with the Messaging 1.1.0 push notification features follow these steps:
+
+#### Step 1: Add the Messaging receivers in the App manifest
+
+Within your App's `AndroidManifest.xml` file, add a [manifest-declared receiver](https://developer.android.com/guide/components/broadcasts#manifest-declared-receivers) inside the application tag to subscribe to broadcast actions sent by the Messaging extension. A second manifest-declared receiver must be added to listen for notification interactions if Messaging handled push notification interaction tracking will be used:
+
+```groovy
+ <!--messaging extension handled push notification broadcast receiver-->
+   <receiver
+      android:name=".NotificationBroadcastReceiver"
+      android:exported="false">
+         <intent-filter>
+	    <action android:name="${applicationId}_adb_action_notification_clicked" />
+            <action android:name="${applicationId}_adb_action_button_clicked" />
+            <action android:name="${applicationId}_adb_action_notification_deleted" />
+            <action android:name="${applicationId}_adb_action_notification_created" />
+            <action android:name="${applicationId}_adb_action_silent_notification_created" />
+	 </intent-filter>
+   </receiver>
+
+<!--messaging extension handled push notification interactions broadcast receiver-->
+   <receiver
+      android:name="com.adobe.marketing.mobile.MessagingPushInteractionHandler"
+      android:exported="false" >
+   </receiver>
+```
+
+{% hint style="info" %}
+
+The intent filters within the first receiver can be added to an existing manifest-declared `BroadcastReceiver` to listen for any notification events sent by the  Messaging extension.
+
+{% endhint %}
+
+#### Step 2: Call the new Messaging push notification creation API
+
+In the `FirebaseMessagingService#onMessageReceived` function of your app, invoke `handlePushNotificationWithRemoteMessage` with the `RemoteMessage` received from Firebase Cloud Messaging. Additionally, a boolean flag enabling Messaging push notification interaction tracking is required when invoking the API.
+
+```java
+public void onMessageReceived(RemoteMessage message) {
+  super.onMessageReceived(message);
+  // Messaging handling of the push payload. If shouldHandleTracking is true then the Messaging extension will handle push notification interaction tracking automatically.
+  Messaging.handlePushNotificationWithRemoteMessage(message, true);
+}
+```
+
+#### Step 3: Add/Update a Broadcast Receiver object to listen for Messaging created push notification broadcasts
+
+The Messaging extension will broadcast events on normal notification creation, silent notification creation, notification deletion, notification click, or notification button presses. A `BroadcastReceiver` [must be declared in the AndroidManifest.xml](#Step 1: Adding the Messaging receivers in the App manifest) and a class subclassing `BroadcastReceiver` must be added to handle the broadcasted events:
```

Comment:
That anchor will not work - it needs to have "-" within the thing.

So it should be (#adding-the-messaging-receivers-in-the-app-manifest).